### ISSUE TITLE
Multisimilarity Loss false return statement 

### ISSUE THREAD

**Issue by twpkevin06222** [Ximilar-com/tf-metric-learning#1]
The call function for Multisimiliarity loss:
```
 def call(self, inputs):       
       embeddings, labels = inputs[ANCHOR], inputs[LABELS]       
       loss = self.loss_fn(embeddings, labels)      
       loss = loss * self.weight
       self.add_loss(loss)        
       self.add_metric(loss, name=self.name, aggregation="mean")        
       return inputs
```
The return should return loss instead of the inputs.

**Comment by Cospel** [comment]
The entire purpose of implementing this as layers is not returning the loss, we are adding loss through self.add_loss and show it in progress bar as self.add_metric.

The advantage is that you can add another layers (and another loss functions).

**Comment by twpkevin06222** [comment]
Thanks for the feedback! I was confused while having a quick glance at the code. Will close this issue.
